src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (a) => {
+    counter++;
+
+    return (b) => {
+      if (counter <= 3) {
+        return a + b;
+      } else {
+        return counter % 2 === 0 ? `Bzzz... Error!` : a + b;
+      }
+    };
+  };
```

Comment:
Don't nest ternary operator inside if. You need to simplify your logic. Return `a + b` in single place.